cluster/addons/fluentd-gcp/fluentd-gcp-configmap.yaml

```diff
@@ -450,15 +453,16 @@ data:
       buffer_type file
       buffer_path /var/log/fluentd-buffers/kubernetes.system.buffer
       buffer_queue_full_action block
-      buffer_chunk_limit 1M
+      buffer_chunk_limit 512k
       buffer_queue_limit 2
       flush_interval 5s
       max_retry_wait 30
       disable_retry_limit
       num_threads 2
       use_grpc true
-      # Use Metadata Agent to get monitored resource.
-      enable_metadata_agent true
```

Comment:
Why are we turning this off? The logging agent will construct the resources itself anyway, but the metadata agent can serve as a fallback for the resources it doesn't know about.

Reply:
Performance. Having this enabled currently results in an extreme performance degradation as opposed to doing everything in-process. This helps get us closer to exceeding the current throughput numbers achievable by the gke_container logging agent. Until we implement performance optimizations, this is a better path forward for our users, as all of the kubernetes resource types are currently fully supported.

Reply:
Fine. Let's re-enable this when those performance improvements are implemented.

Reply:
Could you give a short list of resources, that will stop working with this change?